### ISSUE TITLE
Single-frame panoptic segmentation results on validation set

### ISSUE THREAD

**Issue by minghanz** [MehmetAygun/4D-PLS#12]
Hi, I want to ask whether there are single-frame panoptic segmentation results on validation set available. In the paper there is only test-set result. Without validation set results, there is no reference to compare with, and it is hard to check whether my single-frame model is trained correctly. Thank you!

**Comment by MehmetAygun** [comment]
Hi, unfortunately, I don't have the results for the validation set, but if I remember correctly they should be comparable to test set performance.
